### DISCUSSION TITLE
event on close for modal/flyout

### DISCUSSION THREAD

**Discussion by ultimateBusiness** [livewire/flux#347] [Feature Requests]
It would be great if we can have an event on close (click on button and hit esc-key) to listen for, so we can handle something after closing at the parent level.

**Comment by jeffchown** [comment]
@ultimateBusiness See https://github.com/livewire/flux/discussions/417#discussioncomment-10943294

**Comment by calebporzio** [comment]
Added a close listener in the latest release: https://fluxui.dev/components/modal#close-events
